CONTRIBUTING.md

```diff
@@ -78,6 +78,18 @@ move forward.
   The AsyncAPI spec, despite describing technical behavior, is intended to be
   read by people. Use natural tone and include motivation and examples.
 
+## Breaking Change vs Non Breaking Change
+
+Specifications are not much different than software. Some changes provided in the spec can cause breaking changes for tools that support it. For example, if one of the properties that was `required` becomes `optional`, it is considered a breaking change because some tools might depend on that property and fail if all suddenly it is not provided. 
+
+_Non-breaking changes:_
+- Adding a property
```

Comment:
should this be adding an optional property?

Reply:
Yes, definitely. If you add a required property, it's a breaking change.

Reply:
@dalelane makes sense, can you make a code change suggestion for @Dule-martins please?

Reply:
@derberg I just committed all suggestions made here.